lib/UA.js

```diff
@@ -665,6 +665,16 @@ module.exports = class UA extends EventEmitter
            * and without To tag.
            */
           break;
+        case JsSIP_C.NOTIFY:
+          /**
+           * receive new sip event
+           */
+          this.emit('sipEvent', {
+            event: request.event.event,
```

Comment:
`request.event` provides much also the parameters which are also useful. 

Please change `request.event.event` by `request.event`